lib/hexo/post.js

```diff
@@ -11,24 +11,19 @@ const { copyDir, exists, listDir, mkdirs, readFile, rmdir, unlink, writeFile } =
 const yfm = require('hexo-front-matter');
 
 const preservedKeys = ['title', 'slug', 'path', 'layout', 'date', 'content'];
-
-const _escapeContent = (cache, str) => {
-  const placeholder = '\uFFFC';
-  return `<!--${placeholder}${cache.push(str) - 1}-->`;
-};
+const rEscapeContent = /<escape(?:[^>]*)>([\s\S]*?)<\/escape>/g;
+const rPlaceholder = /(?:<|&lt;)!--hexoPostRenderEscape:(\d+)--(?:>|&gt;)/g;
```

Comment:
Since `\uFFFC` will be rendered by Nunjucks, I have to change it to `hexoPostRenderEscape:`